components/navigation/ui/src/main/resources/XWiki/SearchSuggestConfig.xml

```diff
@@ -526,7 +526,7 @@
       <resultsNumber>5</resultsNumber>
     </property>
     <property>
-      <url>$xwiki.getURL('PhenoTips.SuggestPatientsService', 'get', 'outputSyntax=plain&amp;permission=view')</url>
+      <url>$xcontext.getURLFactory().getServerURL($xcontext.getContext())/rest/patients/suggest?permission=view</url>
```

Comment:
You don't need to specify `$xcontext.getURLFactory().getServerURL($xcontext.getContext())`, use `${request.contextPath}/rest/patients/suggest?permission=view` works.